tests/encapsulate_derived.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -xeuo pipefail
+# Pull the latest FCOS build, unpack its container image, and verify
+# that we can re-encapsulate it as chunked.
+
+ver=37.20230201.20.0
+packaging_ver=37.20221111.20.0
+compare_ver=37.20230131.20.0
+container=quay.io/fedora/fedora-coreos:${ver}
+
+tmpdir=/tmp/coreTest-${ver}
+p_tmpdir=/tmp/coreTest-${packaging_ver}
+mkdir -p ${tmpdir}
```

Comment:
Let's avoid predictable temporary filenames.  Better to use a pattern more like:
```
tmpdir=$(mktemp -d)
cd ${tmpdir}
```